libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -65,7 +65,8 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @Param: AUTO_SWITCH
     // @DisplayName: Automatic Switchover Setting
     // @Description: Automatic switchover to GPS reporting best lock
-    // @Values: 0:Disabled,1:Enabled
+    // @Values: 0:Disabled,1:Hard,2:Soft
+    // @RebootRequired: True
     // @User: Advanced
     AP_GROUPINFO("AUTO_SWITCH", 3, AP_GPS, _auto_switch, 1),
```

Comment:
@OXINARF unfortunately thats a bit orthogonal to the concept of bringing in the behavior disabled until confidence is gained on it across the board. Unless we are that confident in this?